docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -636,7 +647,8 @@ The `isNull` method have been removed, you can replace it with a very basic chec
 
 #### Remove the `mergeDateAndTime` method
 
-The `mergeDateAndTime` method have been removed, you can use the `setHours`, `setMinutes`, and `setSeconds` methods to recreate its behavior:
+The `mergeDateAndTime` method have been removed.
+You can use the `setHours`, `setMinutes`, and `setSeconds` methods to recreate its behavior:
```

Comment:
Me IDE was suggesting me to do it on some longer sentences to split it in two
I applied it everywhere, WDYT?

Reply:
Makes a lot of sense! 👏 👍